### ISSUE TITLE
Help please - Adblock Latitude 3.0.2 will not block Youtube ads

### ISSUE THREAD

**Issue by sagraplus** [Pale-Moon-Addons-Team/adblock-latitude#4]
I have Adblock Latitude 3.0.2 installed and running but it is not blocking ads on Youtube. Can someone help? I could never tolerate a browser/add-on that will not let me block Youtube ads.

I've used Firefox for 20 years but it has become way to bloated and slow. Adblock always worked perfect on Fiirefox. I tried Pale Moon out a week ago and love it's speed.

Also, really weird, if I (using Pale Moon) search something like "HP Computer" in Google and I click on any results on the right side that shows an image of a model with price, etc, nothing happens. I guess that is some sort of Adblock Latitude thing going on???

Thank you for any help!

**Comment by mattatobin** [comment]
Can you tell me what filter sets you are using. Also, were you a previous Adblock Plus user before switching to ABL?

**Comment by sagraplus** [comment]
Hi Matt. Thanks. I have all the boxes checked for "Filter Preferences". I just started using Pale Moon five days ago  and Adblock Latitude is the first adblocker I have tried with it. Matt, actually I haven't noticed any ads on Youtube today, so maybe it got solved. Let me wait a few days to make sure. Then I'll let you know. OK?

Oh, what about that other thing, here?->: "Also, really weird, if I (using Pale Moon) search something like "HP Computer" in Google and I click on any results on the right side that shows an image of a model with price, etc, nothing happens. I guess that is some sort of Adblock Latitude thing going on???"    Thanks again, Bob

**Comment by sagraplus** [comment]
Hi Matt,
 Well, I'm pretty disappointed. I'm still getting ads on videos, including Youtube. Please try to help me solve this. I really hate ads. I don't even watch TV because I hate ads so much. Thank you in advance.
     Bob

**Comment by mattatobin** [comment]
Slow down.. You shouldn't be using EVERY filter list.. Just pick one like just Easylist or Easylist+Chinalist seems to work pretty well.

I currently can't reproduce your trouble as all ads are successfully blocked for me. Can you go to the second tab in filterlists and tell me what you see?

**Comment by sagraplus** [comment]
Hi Matt - thanks - Just tonight I discovered there is a menu for Adblock Latitude under the Tools menu. I found something there called that was checked. I unchecked it and things seem to be OK. Let's see how it goes. Is there a Adblock Latitude user guide or something available? It would be HUGELY helpful to all of us who can't figure out how to use the Filter Preferences for example. Thnaks again, Bob
